kura/org.eclipse.kura.wire.script.filter.provider/src/main/java/org/eclipse/kura/wire/script/filter/provider/OutputWireRecordListWrapper.java

```diff
@@ -19,6 +19,11 @@
 
 import org.eclipse.kura.wire.WireRecord;
 
+/**
+ * 
+ * @deprecated as of Kura 5.3.0
+ */
+@Deprecated
```

Comment:
There's an extra `*` for each of these type of comments. Feel free to ignore this if you don't want to fix it.

```suggestion
/**
 * @deprecated as of Kura 5.3.0
 */
@Deprecated
```

Reply:
Fixed.